e2e/visual_regression/vfolder/vfolder_page.test.ts

```diff
@@ -0,0 +1,59 @@
+import { loginAsVisualRegressionUser2 } from '../../utils/test-util';
+import { expect, test } from '@playwright/test';
+
+test.describe('Vfolder page Visual Regression Test', () => {
+  test.beforeEach(async ({ page }) => {
+    await loginAsVisualRegressionUser2(page);
+    await page.setViewportSize({
+      width: 1500,
+      height: 1000,
+    });
+    await page.getByRole('link', { name: 'Data' }).click();
+    await page.getByText('This storage backend').waitFor();
+  });
```

Comment:
we don't need to have to log in and go to the data page for every test. Can we leave it outside of test.describe?
```suggestion
  test.beforeEach(async ({ page }) => {
    await loginAsVisualRegressionUser2(page);
    await page.setViewportSize({
      width: 1500,
      height: 1000,
    });
    await page.getByRole('link', { name: 'Data' }).click();
    await page.getByText('This storage backend').waitFor();
  });
test.describe('', () => {});
```